examples/purchase-tester/build.gradle

```diff
@@ -68,6 +68,6 @@ dependencies {
     implementation 'com.google.android.material:material:1.6.0'
     implementation "androidx.datastore:datastore-preferences:1.0.0"
 
-    implementation "androidx.navigation:navigation-fragment-ktx:2.4.2"
-    implementation "androidx.navigation:navigation-ui-ktx:2.4.2"
+    implementation "androidx.navigation:navigation-fragment-ktx:2.5.2"
+    implementation "androidx.navigation:navigation-ui-ktx:2.5.2"
```

Comment:
The old version was bringing an extra dependency that was causing duplicated classes issues https://app.circleci.com/pipelines/github/RevenueCat/purchases-android/3336/workflows/1375c72b-ea03-4e15-9174-c4407adf2d49/jobs/8442